cob_description/urdf/cob3_head/head_prosilicas_kinect.urdf.xacro

```diff
@@ -80,7 +80,7 @@
     </xacro:cob_prosilica_v0>
 
     <!-- cam3d -->
-    <xacro:cob_kinect_v0 name="${name}_cam3d" ros_topic="cam3d" parent="${name}_cam_reference_link">
+    <xacro:cob_kinect_v0 name="cam3d" ros_topic="cam3d" parent="${name}_cam_reference_link">
```

Comment:
this will not work for multiple kinects! if topic needs to be the same than link, then please use "${name}_cam3d" for both